posthog/models/property.py

```diff
@@ -152,6 +152,26 @@ def __init__(self, type: PropertyOperatorType, groups: Union[List[Property], Lis
         self.type = type
         self.groups = groups
 
+    def combine_property_groups(
```

Comment:
TODO: tests!

Reply:
Where is this used?

Reply:
Huh, this one isn't!

Tests just for `combine_properties` then, I guess. 😅

Reply:
What does combine even mean, why do we need two different methods for properties vs property groups?